packages/esm-patient-banner-app/src/banner-tags/print-identifier-sticker-modal.component.tsx

```diff
@@ -0,0 +1,168 @@
+        (identifier) => !excludePatientIdentifierCodeTypes?.uuids.includes(identifier.type.coding[0].code),
+      ) ?? [];
+
+    return {
+      address: patient.address,
+      age: age(patient.birthDate),
+      dateOfBirth: patient.birthDate,
+      gender: getGender(patient.gender),
+      id: patient.id,
+      identifiers: [...identifiers],
+      name: patient ? displayName(patient) : '',
+      photo: patient.photo,
+    };
+  }, [excludePatientIdentifierCodeTypes?.uuids, patient, t]);
+
+  const handleBeforeGetContent = useCallback(
+    () =>
+      new Promise<void>((resolve) => {
+        if (patient && headerTitle) {
+          onBeforeGetContentResolve.current = resolve;
+          setIsPrinting(true);
+        }
+      }),
+    [headerTitle, patient],
+  );
+
+  const handleAfterPrint = useCallback(() => {
+    onBeforeGetContentResolve.current = null;
+    setIsPrinting(false);
+    closeModal();
+  }, [closeModal]);
+
+  const handlePrintError = useCallback(
+    (errorLocation, error) => {
+      onBeforeGetContentResolve.current = null;
+
+      showSnackbar({
+        isLowContrast: false,
+        kind: 'error',
+        title: t('printError', 'Print error'),
+        subtitle: t('printErrorExplainer', 'An error occurred in "{{errorLocation}}": ', { errorLocation }) + error,
+      });
+
+      setIsPrinting(false);
+    },
+    [t],
+  );
+
+  const handlePrint = useReactToPrint({
+    content: () => contentToPrintRef.current,
+    documentTitle: `${patientDetails.name} - ${headerTitle}`,
+    onAfterPrint: handleAfterPrint,
+    onBeforeGetContent: handleBeforeGetContent,
+    onPrintError: handlePrintError,
+  });
+
+  return (
+    <>
+      <ModalHeader closeModal={closeModal} title={t('printIdentifierSticker', 'Print identifier sticker')} />
+      <ModalBody>
+        <PrintComponent
+          patientDetails={patientDetails}
+          printIdentifierStickerFields={printIdentifierStickerFields}
+          printIdentifierStickerSize={printIdentifierStickerSize}
+          printRef={contentToPrintRef}
+          t={t}
+        />
+      </ModalBody>
+      <ModalFooter>
+        <Button kind="secondary" onClick={closeModal}>
+          {t('cancel', 'Cancel')}
+        </Button>
+        <Button className={styles.button} disabled={isPrinting} onClick={handlePrint} kind="primary">
+          {isPrinting ? (
+            <InlineLoading className={styles.loader} description={t('printing', 'Printing') + '...'} />
+          ) : (
+            t('print', 'Print')
+          )}
+        </Button>
+      </ModalFooter>
+    </>
+  );
+};
+
+const PrintComponent = ({ patientDetails, printIdentifierStickerFields, printIdentifierStickerSize, printRef, t }) => {
+  return (
+    <div className={styles.stickerContainer} ref={printRef}>
+      <style type="text/css" media="print">
+        {`
+          @page {
+            size: ${printIdentifierStickerSize};
+          }
+        `}
+      </style>
```

Comment:
@ibacher it's trivial to add a config property that governs which orientation to default to and then it would leverage [this approach](https://github.com/MatthewHerbst/react-to-print#set-the-page-orientation).

Reply:
Scratch that, it appears that it's not so straightforward to change the orientation for custom sizes (like the default `4in 6in` value). For standard [page sizes](https://developer.mozilla.org/en-US/docs/Web/CSS/%40page/size#page-size), something like this works well:

```tsx
const  { printPageOrientation } = useConfig();

<style type="text/css" media="print">
  {`
    @page {
      size: A3 ${printPageOrientation};
    }
  `}
</style>
```

Reply:
It looks like this is something else worth looking into when evaluating whether to move away from this library. I'm proceeding to merge to unblock Madiro for now.